Sources/Model/ObjC/Common/Core Data/TracksContextManager.m

```diff
@@ -49,7 +49,7 @@ - (NSPersistentStoreCoordinator *)persistentStoreCoordinator {
         if (![_persistentStoreCoordinator addPersistentStoreWithType:NSSQLiteStoreType configuration:nil URL:storeURL options:nil error:&error]) {
             // Replace this with code to handle the error appropriately.
             // abort() causes the application to generate a crash log and terminate. You should not use this function in a shipping application, although it may be useful during development.
-            TracksLogError(@"Unresolved error %@, %@", error, [error userInfo]);
+            TracksLogPanic(@"[Tracks] PersistentStore Error: %@, %@", error, [error userInfo]);
```

Comment:
See the Day One PR for my comment about naming this differently.

Reply:
Apologies!! couldn't find anything related to this. To which PR you refer to? (checked on [this one](https://github.com/bloom/DayOne-iOS/pull/17932) but has no logging comments?).

Thank you!!

Reply:
Ah, this comment: https://github.com/bloom/DayOne-iOS/pull/17932#discussion_r766954441. 
I just meant naming it `logToSentry` or `logToErrorReportingSystem` or something like that instead of `TracksLogPanic`

Reply:
Eh, I've changed my mind, and I kind of like the panic name. That is basically what is going on if you call this.